### PR TITLE
Right margin wrapping protobuf plugin

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**
This PR contains the changes in wrapping for 100 column right margin.

### CODE REVIEW

**Comment by alexander-aleksandrov** [comment]
@dmytro-grankin PTAL

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/proto/ProtoToJavaTypeMapper.java [hunk 1]
```diff
@@ -182,8 +191,13 @@ private String getTypeUrl(String typeName, Collection<String> parentMsgNames) {
...
     private String getJavaClassName(String typeName, Collection<String> parentTypeNames) {
-        final String parentClassesPrefix = getParentTypesPrefix(parentTypeNames, JAVA_INNER_CLASS_SEPARATOR);
-        final String result = javaPackagePrefix + commonOuterClassPrefix + parentClassesPrefix + typeName;
+        final String parentClassesPrefix = getParentTypesPrefix(parentTypeNames,
+                                                                JAVA_INNER_CLASS_SEPARATOR);
+        final String result =
+                javaPackagePrefix +
+                commonOuterClassPrefix +
+                        parentClassesPrefix +
+                        typeName;
```
Comment (dmytro-grankin) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/proto/ProtoToJavaTypeMapper.java]: Same as above.

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/DescriptorSetUtil.java [hunk 2]
```diff
@@ -90,7 +93,8 @@ private DescriptorSetUtil() {
                 }
             }
         } catch (@SuppressWarnings("OverlyBroadCatchBlock") IOException e) {
-            throw new RuntimeException("Cannot get proto file descriptors. Path = " + descriptorSetPath, e);
+            throw new RuntimeException("Cannot get proto file descriptors. Path = " +
+                                               descriptorSetPath, e);
```
Comment (dmytro-grankin) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/DescriptorSetUtil.java]: Message extracting will increase readability.

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/PropertiesWriter.java [hunk 3]
```diff
@@ -107,7 +108,8 @@ private static void prepareTargetFile(Properties props, File file) {
                 final FileInputStream fis = new FileInputStream(file);
                 props.load(fis);
             } catch (@SuppressWarnings("OverlyBroadCatchBlock") IOException e) {
-                throw new RuntimeException("Error loading the properties from the file: " + file.getAbsolutePath(), e);
+                throw new RuntimeException("Error loading the properties from the file: " +
+                                                   file.getAbsolutePath(), e);
```
Comment (dmytro-grankin) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/PropertiesWriter.java]: Same as above.

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/PropertiesWriter.java [hunk 4]
```diff
@@ -118,26 +120,29 @@ private static void createParentFolders(File file) {
         try {
             Files.createParentDirs(file);
         } catch (IOException e) {
-            throw new RuntimeException("Cannot create the parent folders at " + file.getAbsolutePath(), e);
+            throw new RuntimeException("Cannot create the parent folders at " +
+                                               file.getAbsolutePath(), e);
```
Comment (dmytro-grankin) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/PropertiesWriter.java]: Same as above.

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/enrichments/EnrichmentsFinder.java [hunk 5]
```diff
@@ -356,15 +367,21 @@ private static void put(Map.Entry<String, String> entry, Multimap<String, String
 
     @SuppressWarnings("DuplicateStringLiteralInspection")
     private static IllegalStateException invalidByOptionValue(String msgName) {
-        throw new IllegalStateException("Field of message `" + msgName + "` has invalid 'by' option value, " +
+        throw new IllegalStateException("Field of message `" +
+                                                msgName +
+                                                "` has invalid 'by' option value, " +
```
Comment (alexander-aleksandrov) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/enrichments/EnrichmentsFinder.java]: Done

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/enrichments/EnrichmentsFinder.java [hunk 6]
```diff
@@ -356,15 +367,21 @@ private static void put(Map.Entry<String, String> entry, Multimap<String, String
...
 
     @SuppressWarnings("DuplicateStringLiteralInspection")
     private static IllegalStateException invalidByOptionUsage(String msgName) {
-        throw new IllegalStateException("Field of message `" + msgName + "` has invalid 'by' option value. " +
-                                                "Wildcard type is not allowed with multiple arguments. " +
-                                                "Please, specify the type either with `by` or with `enrichment_for` annotation.");
+        throw new IllegalStateException(
+                "Field of message `" +
+                        msgName +
+                        "` has invalid 'by' option value. " +
```
Comment (alexander-aleksandrov) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/enrichments/EnrichmentsFinder.java]: Done

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/proto/ProtoToJavaMapperPlugin.java [hunk 7]
```diff
@@ -71,26 +72,33 @@
...
         final Action<Task> testScopeAction = testScopeActionFor(project);
-        logDependingTask(log(), MAP_TEST_PROTO_TO_JAVA, PROCESS_TEST_RESOURCES, GENERATE_TEST_PROTO);
-        final GradleTask testScopeTask = newTask(MAP_TEST_PROTO_TO_JAVA,
-                                                 testScopeAction).insertAfterTask(GENERATE_TEST_PROTO)
-                                                                 .insertBeforeTask(PROCESS_TEST_RESOURCES)
-                                                                 .applyNowTo(project);
+        logDependingTask(log(),
+                         MAP_TEST_PROTO_TO_JAVA,
+                         PROCESS_TEST_RESOURCES,
+                         GENERATE_TEST_PROTO);
```
Comment (alexander-aleksandrov) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/proto/ProtoToJavaMapperPlugin.java]: Done

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/proto/ProtoToJavaTypeMapper.java (hunk 1 above)
Comment (alexander-aleksandrov) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/proto/ProtoToJavaTypeMapper.java]: Done

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/DescriptorSetUtil.java (hunk 2 above)
Comment (alexander-aleksandrov) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/DescriptorSetUtil.java]: Done

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/PropertiesWriter.java (hunk 3 above)
Comment (alexander-aleksandrov) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/PropertiesWriter.java]: Done

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/PropertiesWriter.java (hunk 4 above)
Comment (alexander-aleksandrov) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/PropertiesWriter.java]: Done

**Comment by alexander-aleksandrov** [comment]
@dmytro-grankin PTAL

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/proto/ProtoToJavaMapperPlugin.java [hunk 8]
```diff
@@ -78,10 +78,7 @@ public void apply(final Project project) {
                                                            .applyNowTo(project);
 
         final Action<Task> testScopeAction = testScopeActionFor(project);
-        logDependingTask(log(),
-                         MAP_TEST_PROTO_TO_JAVA,
-                         PROCESS_TEST_RESOURCES,
-                         GENERATE_TEST_PROTO);
+        logDependingTask(log(),MAP_TEST_PROTO_TO_JAVA, PROCESS_TEST_RESOURCES, GENERATE_TEST_PROTO);
```
Comment (dmytro-grankin) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/proto/ProtoToJavaMapperPlugin.java]: There is no space after `log()` and the line is a little bit longer than 100 symbols.

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/DescriptorSetUtil.java [hunk 9]
```diff
@@ -93,8 +91,8 @@ private DescriptorSetUtil() {
                 }
             }
         } catch (@SuppressWarnings("OverlyBroadCatchBlock") IOException e) {
-            throw new RuntimeException("Cannot get proto file descriptors. Path = " +
-                                               descriptorSetPath, e);
+            final String errMsg = "Cannot get proto file descriptors. Path = ";
+            throw new RuntimeException(errMsg + descriptorSetPath, e);
```
Comment (dmytro-grankin) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/DescriptorSetUtil.java]: `errMsg = "... Path = " + descriptorSetPath;` more elegant. Do like this if there is enough space. Apply to the similar changes also.

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/proto/ProtoToJavaMapperPlugin.java (hunk 8 above)
Comment (alexander-aleksandrov) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/lookup/proto/ProtoToJavaMapperPlugin.java]: Done

File: protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/DescriptorSetUtil.java (hunk 9 above)
Comment (alexander-aleksandrov) [inline protobuf-plugin/plugin/src/main/java/org/spine3/gradle/protobuf/util/DescriptorSetUtil.java]: Done

**Comment by alexander-aleksandrov** [comment]
@dmytro-grankin PTAL

**Pull request merged**